ntfy/terminal.py

```diff
@@ -10,9 +11,10 @@ def get_tty():
 
 
 def linux_window_is_focused():
-    window_id = int(check_output(['xprop', '-root', '\t$0',
-                                  '_NET_ACTIVE_WINDOW']).split()[1], 16)
-    return int(environ['WINDOWID']) == window_id
+    xprop_cmd = shlex.split("xprop -root _NET_ACTIVE_WINDOW")
```

Comment:
Any particular reason for using shlex and splitting a str command instead of defining it as a list to begin with?